django_recaptcha/static/django_recaptcha/js/widget_v3.js

```diff
@@ -0,0 +1,50 @@
+(function () {
+
+  function main() {
+    grecaptcha.ready(function () {
+      for (const captchaElement of document.querySelectorAll(".g-recaptcha")) {
+
+        const widgetUuid = captchaElement.getAttribute("data-widget-uuid");
+        if (!widgetUuid) {
+          console.warn("reCAPTCHA widget with missing UUID");
+          break;
+        }
+        console.log(`found reCAPTCHA widget with UUID '${widgetUuid}'`);
+
+        const formElement = captchaElement.closest("form");
+        if (!formElement) {
+          console.warn("reCAPTCHA widget with UUID '" + widgetUuid + "' is not part of a form");
+          break;
+        }
+
+        const publicKey = captchaElement.getAttribute("data-sitekey");
+        if (publicKey === null) {
+          console.warn("public key missing missing from reCAPTCHA widget with UUID '" + widgetUuid + "'");
+          break;
```

Comment:
**Suggestion:** please use template literals for string interpolation for consistency with the other scripts.